### ISSUE TITLE
在线演示的项目有问题

### ISSUE THREAD

**Issue by ulanbator** [HongqingCao/GitDataV#14]
根据提供的在线演示项目，输入用户名的进去显示用户名not found。

**Comment by HongqingCao** [comment]
若是你自己的github，请输入对应的用户名是：ulanbator

**Comment by ulanbator** [comment]
对 我输入我的用户名办的错 输入你的用户名也报错 您可以试试。------------------ 原始邮件 ------------------
发件人: "codercao"<notifications@github.com>
发送时间: 2020年3月7日(星期六) 下午5:30
收件人: "HongqingCao/GitDataV"<GitDataV@noreply.github.com>;
抄送: "fangzheng"<1742133696@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [HongqingCao/GitDataV] 在线演示的项目有问题 (#14)



若是你自己的github，请输入对应的用户名是：ulanbator
 
&mdash;

**Comment by HongqingCao** [comment]
已正常通过输入你github用户名ulanbator打开可视化页面。

**Comment by ulanbator** [comment]
谢谢你------------------ 原始邮件 ------------------
发件人: "codercao"<notifications@github.com>
发送时间: 2020年3月7日(星期六) 下午5:57
收件人: "HongqingCao/GitDataV"<GitDataV@noreply.github.com>;
抄送: "fangzheng"<1742133696@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [HongqingCao/GitDataV] 在线演示的项目有问题 (#14)



已正常通过输入你github用户名ulanbator打开可视化页面。
 
&mdash;
